assets/scss/main.scss

```diff
@@ -51,3 +51,10 @@
 }
 
 @import "styles_project";
+
+// Fix tables visual bug
```

Comment:
question: which visual bug is this solving? I can't seem to be seeing a difference with/without :(

Reply:
![screencapture-deploy-preview-628-cert-manager-netlify-app-docs-concepts-certificaterequest-2021-06-28-12_48_43](https://user-images.githubusercontent.com/42113979/123624727-36c63300-d80f-11eb-92ac-d33376095471.png)
Now the table is not wider than the rest of the content. Isn't that the issue you were describing here?: https://github.com/cert-manager/website/pull/628#issuecomment-867972867

ref. https://deploy-preview-628--cert-manager.netlify.app/docs/concepts/certificaterequest/

Reply:
I realize that I misled you when taking about a CSS bug in tables 😅

I was thinking about the tables in the release process page. The tables don't look the same when they are inside an ordered list. `master` and c27168d look the same w.r.t those tables:

<img alt="master" src="https://user-images.githubusercontent.com/2195781/123639655-b6103280-d820-11eb-850c-957c59778975.png" width="300"><img alt="c27168d" src="https://user-images.githubusercontent.com/2195781/123639117-2cf8fb80-d820-11eb-8b01-410cae9d6fe9.png" width="300">

Reply:
I suggest that we leave these cosmetic changes to an ulterior PR, what do you think?

Reply:
I'll revert that change.